dataactcore/utils/duns.py

```diff
@@ -35,24 +35,29 @@ def get_client(ssh_key=None):
     sam_config = CONFIG_BROKER.get('sam_duns')
     if not sam_config:
         return None
-    host = sam_config.get('host') if not ssh_key else sam_config.get('host_ssh')
-    port = sam_config.get('port')
-    username = sam_config.get('username')
-    password = sam_config.get('password')
-    pkey = None
+
     if ssh_key:
-        with RetrieveFileFromUri(ssh_key, binary_data=False).get_file_object() as key_obj:
-            pkey = paramiko.RSAKey.from_private_key(key_obj, password=password)
+        host = sam_config.get('host_ssh')
+        username = sam_config.get('username_ssh')
+        password = sam_config.get('password_ssh')
+
+        ssh_key_file = RetrieveFileFromUri(ssh_key, binary_data=False).get_file_object()
```

Comment:
Pretty sure this still needs the `with`.

Reply:
This was changed to clarify that it's getting the key file in one call and using `ssh_key_password` with it in the following call.